maps/yw/cryogaia_areas.dm

```diff
@@ -345,6 +345,10 @@
 	name = "\improper Upper Southern Level Hallways"
 	icon_state = "hallC1"
 
+/area/cryogaia/station/hallway/primary/upper/southeast
+	name = "\improper South Central Storage Hallway"
+	icon_state = "hallC1"
+
```

Comment:
```suggestion
```
Can remain in this file but should not remain in the exploration subsection of it and the path be better if it was /area/hallway/primary/southeast as upper refers to the top level of the station.